enterprise/coderd/groups.go

```diff
@@ -250,7 +276,10 @@ func (api *API) patchGroup(rw http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	patchedMembers, err := api.Database.GetGroupMembers(ctx, group.ID)
+	patchedMembers, err := api.Database.GetGroupMembers(ctx, database.GetGroupMembersParams{
+		ID:             group.ID,
+		OrganizationID: group.OrganizationID,
+	})
```

Comment:
If we change the SQL to what I mentioned earlier, we can just pass the `group.ID`